README.md

```diff
@@ -89,6 +89,10 @@ non-WGSL shaders if you're running on WebGPU.
 
 WGSL is always supported by default, but GLSL and SPIR-V need features enabled to compile in support.
 
+Note that the WGSL specification is still under development,
+so the [draft specification][wgsl spec] does not exactly describe what `wgpu` supports.
+See [below](https://github.com/jimblandy/wgpu/tree/doc-wgsl-version#tracking-the-webgpu-and-wgsl-draft-specifications) for details.
```

Comment:
That's probably not what you want to do in the WGSL repo. In my MR I linked in a way that should still work regardless or current repo.

Reply:
Oh, you're right. I was careless. Thanks for catching this.

I also didn't actually intend to merge this before cwfitzgerald's review.

Reply:
Fixed in 94ce7639.